stream_alert_cli/logger.py

```diff
@@ -33,3 +54,14 @@
     if logger.startswith('StreamAlert'):
         continue
     logging.getLogger(logger).setLevel(logging.CRITICAL)
+
+# Get a logging MemoryHandler with a default buffer size of 1000
+# We don't care about assigning a target to this handler since these logs
+# will not actually be written out to disk, etc
+LOG_ERROR_HANDLER = logging.handlers.MemoryHandler(1000)
+
+# Add a filter to suppress everything that is not an error
+LOG_ERROR_HANDLER.addFilter(SuppressNonErrors())
+
+# Add the MemoryHandler to the root logger to capture all logs in all loggers
+logging.getLogger().addHandler(LOG_ERROR_HANDLER)
```

Comment:
This isn't unit tests and doesn't get added for all streamalert operations - this happens only in the CLI package - but you are right I can update to only happen if integration tests are being run! making that change